.pylintrc

```diff
@@ -142,13 +142,14 @@ confidence=HIGH,
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use "--disable=all --enable=classes
 # --disable=W".
-disable=raw-checker-failed,
+disable=abstract-method,
         bad-inline-option,
-        locally-disabled,
+        deprecated-pragma,
         file-ignored,
+        locally-disabled,
+        raw-checker-failed,
```

Comment:
Just reordered and added the exclussion of `abstract-method`...(false positives).